package.json

```diff
@@ -29,15 +29,11 @@
     "@apollo/client": "^3.5.10",
     "@gorhom/bottom-sheet": "^4.1.5",
     "@polkadot/api": "7.9.1",
-    "@polkadot/apps-config": "^0.103.1",
     "@polkadot/reactnative-identicon": "0.89.2",
-    "@polkadot/ui-settings": "^0.89.2",
     "@react-native-community/async-storage": "^1.12.1",
     "@react-native-community/clipboard": "^1.5.1",
-    "@react-native-community/masked-view": "^0.1.11",
     "@react-native-firebase/app": "^14.7.0",
     "@react-native-firebase/messaging": "^14.7.0",
-    "@react-native-firebase/perf": "^14.7.0",
```

Comment:
This should not be removed. I think it is needed for the new performance monitoring set-up we have.